sdk/src/claim.rs

```diff
@@ -980,10 +1055,16 @@ impl Claim {
         data_hashes
     }
 
+    pub fn bmff_hash_assertions_mut(&self) -> Vec<&Assertion> {
```

Comment:
I don't see anything mutable here. Why the `_mut` suffix?

Reply:
Looks like I changed the usage.  I will fix the name.